crates/apub_lib/Cargo.toml

```diff
@@ -14,6 +14,7 @@ chrono = "0.4.19"
 serde = { version = "1.0.136", features = ["derive"] }
 async-trait = "0.1.53"
 url = { version = "2.2.2", features = ["serde"] }
+serde_json = { version = "1.0.79", features = ["preserve_order"] }
```

Comment:
Removed this in #2239, but its used in a doc test. Not sure why ci for that pr passed anyway.

Reply:
Yes but it should have failed during the cargo test run in ci.